javascript/javascript1/week3/homework2.js

```diff
@@ -0,0 +1,24 @@
+// When will we be there??
+
+// The estimated time of arrival (ETA) is the time when a ship,
+// vehicle, aircraft, cargo, person, or emergency service is expected to arrive at a certain place.
+
+// Write a function where you speficy your speed in km/h and how far
+// you have to go in km. The function has to return the time it will take to arrive at ¨
+// your destination. The time should be formatted like this: 3 hours and 34 minutes.
+
+// Hint: For formatting your best friend is Google!
+
+const travelInformation = {
+  speed: 50,
+  destinationDistance: 432,
+};
+
+const timeToArrive = (obj) => {
+  let result = obj.destinationDistance / obj.speed;
+  let minutes = (result - parseInt(result)) * 60;
+  return `${Math.floor(result)} hours and ${minutes.toFixed()} minutes`;
+};
```

Comment:
The function returns a message, not just `timeToArrive`. In that sense `timeToArriveMessage` could be a better name, and then `result` could be changed to `time`.
Both, `result` and `minutes` can be set as `const`.

Reply:
corrected!